library/src/test/java/com/basgeekball/awesomevalidation/ValidationHolderTest.java

```diff
@@ -44,13 +47,16 @@ protected void setUp() throws Exception {
         mMockConfirmationTextInputLayout = mock(TextInputLayout.class);
         Pattern mockPattern = PowerMock.createMock(Pattern.class);
         NumericRange mockNumericRange = mock(NumericRange.class);
+        CustomValidation customValidation = mock(CustomValidation.class);
```

Comment:
Should follow the naming convention: `mockCustomValidation` instead of `customValidation`.